test/controller/confirm.company.controller.spec.unit.ts

```diff
@@ -256,13 +256,14 @@ describe("confirm company tests", () => {
     expect(response.text).toContain("14 April 2015");
   });
 
-  it("should correctly display the latest GAZ1 date from the filing history if company in gaz1, with gaz2 requested", async () => {
+  it("should correctly display the notice expired message from the filing history if company in gaz1, with gaz2 requested", async () => {
 
     const mockValidAccessToken = retrieveAccessTokenFromSession as jest.Mock;
     mockValidAccessToken.mockReset().mockImplementation(() => ACCESS_TOKEN);
 
     mockGetObjectionSessionValue.mockReset().mockImplementation(() => dummyCompanyProfile);
 
+    mockGetCompanyEligibility.mockReset();
```

Comment:
you shouldn't need to add the mock reset as it's happening also in the next line, so it is being Reset twice.